### ISSUE TITLE
can the disk serials recover when reboot the computer?

### ISSUE THREAD

**Issue by eway244** [SamuelTulach/mutante#15]
can the disk serials recover when reboot the computer?
in another way, is it temporally  spoofed

**Comment by Relentless0x7** [comment]
yes this is a temp spoofer, after restarting you will be back to normal. it will require another spoof to play again.
